test/debug.js

```diff
@@ -99,7 +99,9 @@ describe("Debug", function() {
 
         assert.equal(lastop.op, "STOP");
         assert.equal(lastop.gasCost, 1);
-        assert.equal(lastop.pc, 131);
+        assert.equal(lastop.pc, 141);
+        assert.equal(lastop.storage['0x0000000000000000000000000000000000000000000000000000000000000000'], '0x000000000000000000000000000000000000000000000000000000000000001a')
+        assert.equal(lastop.storage['0x0000000000000000000000000000000000000000000000000000000000000001'], '0x000000000000000000000000000000000000000000000000000000000000001f')
```

Comment:
I believe the trace reports values without leading `0x` for `memory` and `stack` – I can't tell from this PR if that behavior is preserved here or not (test suggests maybe no?)